exercises/atbash-cipher/example.ml

```diff
@@ -0,0 +1,57 @@
+let substitute = function
+  | 'a' .. 'z' as c ->
+    let offset = (Char.code c) - (Char.code 'a') in
+    let code = (Char.code 'z') - offset in
+    Char.chr code
+  | other -> other
+
+let is_encodable = function
+  | 'a' .. 'z' | '0' .. '9' -> true
+  | _ -> false
+
+let explode s =
```

Comment:
There a good summary of the benefits of type signatures on stackoverflow
http://stackoverflow.com/questions/23401698/what-is-a-good-reason-to-use-a-type-signature-for-functions-when-the-compiler-ca